.gitignore

```diff
@@ -1,16 +1,22 @@
-*.vcf.gz
-*.vcf.gz.tbi
+# *.vcf.gz
+# *.vcf.gz.tbi
+# *.bam
+# *.bai
```

Comment:
Why are you removing bam and bai files? These can potentially get really large so it would be good to preserve that in the gitignore